server/app.py

```diff
@@ -5,26 +5,35 @@
 from api.login_handler import LoginResource
 from api.user_handler import UserResource
 from api.meal_item_handler import MealItemResource
+from api.LogoutResource import LogoutResource
+from api.TokenRefreshResource import TokenRefreshResource
 
-from api.ping_handler import ping_handler
-from api.home_handler import home_handler
-
-from flask_jwt_extended import JWTManager
 
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    jwt_refresh_token_required, create_refresh_token,
+    get_jwt_identity
+)
 from models import db
 
 
 def create_app():
     app = Flask(__name__)
+    app.config['JWT_TOKEN_LOCATION'] = ['cookies']
+
+    # Only allow JWT cookies to be sent over https. In production, this
+    # should likely be True
+    app.config['JWT_COOKIE_SECURE'] = False
+    app.config['JWT_ACCESS_COOKIE_PATH'] = '/'
+    app.config['JWT_REFRESH_COOKIE_PATH'] = '/token/refresh'
+    app.config['JWT_COOKIE_CSRF_PROTECT'] = True
+    app.config['JWT_CSRF_IN_COOKIES'] = True
     app.config['JWT_SECRET_KEY'] = 'team-bluejay'
     jwt = JWTManager(app)
 
-    app.register_blueprint(home_handler)
-    app.register_blueprint(ping_handler)
-
     # Database variables
     user = 'postgres'
-    pw = '123456'
+    pw = 'P@ssw0rd'
```

Comment:
Do we have an icebox ticket to move these to environment variables?

Reply:
created it